sources/pug/_config.pug

```diff
@@ -94,4 +94,13 @@
 
     // Predefined variables
     breadcrumbs = [],
-    pageName = 'Home'
+    pageName = 'Home',
+
+    // Meta tags
+    ogData = {
+      title: 'AsyncAPI Online Conference',
+      image: '/images/meta/og-image.jpg',
+      description: 'Join us at the AsyncAPI Online Conference and learn about Open-Source and Event-Driven Architectures.',
+      twitterCard: 'summary_large_image',
+      twitterUrl: 'https://twitter.com/asyncapispec'
+    }
```

Comment:
In slack it works 👏🏼  But I don't know why, but there are still that the same changes as in master repo 🤔  Also as I see you have conflicts, so you should fix them, and then that changes should dissapear :)

![image](https://user-images.githubusercontent.com/20404945/140526117-1aaef1b8-7468-4ce3-9542-e484388928db.png)


![image](https://user-images.githubusercontent.com/20404945/140525907-9ddb213b-a7df-49a4-be27-1b29d86a720b.png)

Reply:
@mcturco I think you added new image version to old branch you already had, but it was merged already, thus conflicts, you will have to open new branch from latest master in your fork and then new PR

Reply:
Oh shoot okay, my bad

Reply:
@magicmatatjahu @derberg opened a new PR here: https://github.com/asyncapi/conference-website/pull/73 sorry about that guys!

Reply:
@mcturco no worries!